internal/renderers/femtovg/lib.rs

```diff
@@ -314,6 +319,10 @@ impl FemtoVGRenderer {
             self.with_graphics_api(|api| callback.notify(RenderingState::AfterRendering, &api))?;
         }
 
+        if let Some(pre_present_callback) = pre_present_callback.take() {
```

Comment:
I don't think you need the take

```suggestion
        if let Some(pre_present_callback) = pre_present_callback {
```

Reply:
Good idea, will do that. Not sure why I did :)